openapi/src/parts/workspace.yaml

```diff
@@ -479,6 +479,12 @@ components:
         description:
           description: A description of the workspace
           type: string
+        removeAttributes:
```

Comment:
I am thinking we should make a separate endpoint for the policy update. If we stick with this approach, we need a way to communicate that the update mode only applies to the policy inputs. It seems weird that you could update the description, but DRY_RUN the policy change.

Also, how does the caller get the DRY_RUN results back? The UI will want to tell the user, you cannot change that because policy XX from object UUID says YY.